ui/app/mainui/Popups.qml

```diff
@@ -338,7 +327,7 @@ QtObject {
                 profileStore: rootStore.profileSectionStore.profileStore
                 contactsStore: rootStore.profileSectionStore.contactsStore
                 communitiesModel: rootStore.profileSectionStore.communitiesList
-
+                destroyOnClose: false
```

Comment:
I wonder... why not here then?

Reply:
in this case needs to do stuff before closing so the destroy is called inside onClose:
```

                onClosed: {
                    if (profilePopup.parentPopup) {
                        profilePopup.parentPopup.close()
                    }
                    destroy()
                }
```